### PR TITLE
drivers: charger: add charger prefix for bq24190

### PR DESCRIPTION
**Pull request by bbilas**
Many (or almost all) drivers contain the specified prefixes related to the driver subsys so add the missing one for the BQ24190.

### CODE REVIEW

File: drivers/charger/charger_bq24190.c [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 #include <errno.h>
 
-#include "bq24190.h"
+#include "charger_bq24190.h"
```
Comment (gmarull) [inline drivers/charger/charger_bq24190.c:11]: I find this one redundant: `drivers/charger/charger_`

Comment (bbilas) [reply drivers/charger/charger_bq24190.c:11]: So in case of new drivers, should we drop that prefix as well?

Comment (gmarull) [reply drivers/charger/charger_bq24190.c:11]: I’d personally do, folder already _namespaces_ here

Comment (bbilas) [reply drivers/charger/charger_bq24190.c:11]: Let's wait a while and find out others' opinions.

**Comment by bbilas** [comment]
@rriveramcrus please take a look.

File: drivers/charger/charger_bq24190.c (hunk 1 above)
Comment (rriveramcrus) [reply drivers/charger/charger_bq24190.c:11]: I see and understand @gmarull's point here. If it is the norm, than this change should be withheld.

Comment (bbilas) [reply drivers/charger/charger_bq24190.c]: Alright.

**Review by rriveramcrus — APPROVED** [review]

**Review by aasinclair — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Pull request merged**